### PR TITLE
CSPL-2388 - Fix helm int tests

### PR DESCRIPTION
**Pull request by akondur**
Fix helm int tests [failure](https://github.com/splunk/splunk-operator/actions/runs/4997225731/jobs/8951485896) caused by the [PR](https://github.com/splunk/splunk-operator/pull/1147).

- Extended the timeout for the helm test suite
- Passed the `TEST_VPC_ENDPOINT_URL` string as an env variable to be picked up by `c3_with_apps_private_link` helm test case.
- Fixed the `m4-with-operator` and `m4` helm test cases to include the right regions(`us-west-2c` was not annotated on the nodes, switched to `us-west-2d` instead).
- Modified the regular expression string to extract region for appFramework
- Added a timeout for aws, azure, minio clients for appFramework
- Upgraded eksctl version

### CODE REVIEW

File: .github/workflows/helm-test-workflow.yml [hunk 1]
```diff
@@ -69,6 +70,7 @@ jobs:
       DEPLOYMENT_TYPE: "helm"
       HELM_REPO_PATH: "../../../../helm-chart"
       INSTALL_OPERATOR: "true"
+      TEST_VPC_ENDPOINT_URL: ${{ secrets.TEST_VPC_ENDPOINT_URL }}
```
Comment (gaurav-splunk) [inline .github/workflows/helm-test-workflow.yml:73]: Can we instead use env variable as part of `.env` file in our project? Just don't want to fall into the trap again where we have to update the secrets again and we dont know why the tests failed when someone changed the secrets or something like that.

Comment (akondur) [reply .github/workflows/helm-test-workflow.yml:73]: @vivekr-splunk and I discussed it. I proposed using this method in order to not have a VPC endpoint on an open sourced repo. In the future we can store these in a separate repo(gitlab?).

Comment (akondur) [reply .github/workflows/helm-test-workflow.yml:73]: If the env method is preferred I can switch to that as well :)

**Review by gaurav-splunk — APPROVED** [review]

File: .github/workflows/helm-test-workflow.yml [hunk 2]
```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - develop
       - main
+      - cspl_2388
```
Comment (gaurav-splunk) [inline .github/workflows/helm-test-workflow.yml:7]: Lets remove this if helm tests are passing now.

**Comment by akondur** [comment]
Successful test runs:

[Helm tests](https://github.com/splunk/splunk-operator/actions/runs/5132948107/jobs/9234907754?pr=1165)
[Int tests](https://github.com/splunk/splunk-operator/actions/runs/5132948112/jobs/9234907745?pr=1165)
[Smoke tests](https://github.com/splunk/splunk-operator/actions/runs/5132948111/jobs/9235464867?pr=1165)

**Review by vivekr-splunk — APPROVED** [review]

**Pull request merged**